src/tag.styles.ts

```diff
@@ -34,23 +34,30 @@ export default [
       color: var(--glide-core-color-static-text-default);
       display: flex;
       font-family: var(--glide-core-typography-family-primary);
-      font-size: var(--glide-core-typography-size-body-small);
       font-weight: var(--glide-core-typography-weight-regular);
       justify-content: center;
       line-height: 1;
       max-inline-size: max-content;
-      min-block-size: var(--glide-core-spacing-base-md);
       opacity: 1;
-      padding: var(--glide-core-spacing-base-xxxs)
```

Comment:
Moved this and the ones above  to `.medium` given that's the size they apply to.